engines/tinsel/actors.cpp

```diff
@@ -226,7 +226,7 @@ void Actor::StartActor(const ACTORDATA *ad, bool bRunScript) {
  * @param bRunScript	Flag for whether to run actor scene scripts
  */
 void Actor::StartTaggedActors(SCNHANDLE ah, int numActors, bool bRunScript) {
-	if (!TinselV2) {
+	if (TinselVersion < TINSEL_V2) {
```

Comment:
just a minor suggestion: semantically it would imo be better to have `if (TinselVersion <= TINSEL_V1) {` here - would make searching for v1 stuff maybe a little bit easier.

Reply:
Good point, done.